packages/@atjson/editor/src/index.ts

```diff
@@ -1,184 +1,168 @@
-import { HIR } from '@atjson/hir';
 import Document from '@atjson/document';
+import WebComponentRenderer from './webcomponent-renderer';
```

Comment:
Yes - I'd started down the road of making it a separate renderer, but kept it close because I wasn't sure where to put the components. I think it could be moved now, will take a look at doing that.